strapi/strapi
# [Marketplace] Add compatibility tooltip

### What does it do?

- Adds a compatibility tooltip with a message for:
  - Plugin not compatible with current version of Strapi
  - Unable to verify if plugin is compatible with current version of Strapi

- Removes react-copy-clipboard and just handle using the native clipboard API (tested on Chrome, Firefox, and Safari)

### How to test it?

- Go to the marketplace
- Look for "Awesome SEO", the button should be disabled and it should have a tooltip on hover with message stating it isn't compatible with the current version of Strapi
- Go to any plugin marked as "Contact developer" in airtable, such as BigBlueButton, it should have a button with a tooltip on hover stating unable to verify compatibility with current version of Strapi. The button should still be clickable
